vehicle/raw_vehicle_cmd_converter/include/raw_vehicle_cmd_converter/node.hpp

```diff
@@ -22,24 +22,25 @@
 
 #include <rclcpp/rclcpp.hpp>
 
-#include <autoware_control_msgs/msg/control_command_stamped.hpp>
+#include <autoware_auto_control_msgs/msg/ackermann_control_command.hpp>
 #include <autoware_debug_msgs/msg/float32_multi_array_stamped.hpp>
 #include <autoware_vehicle_msgs/msg/actuation_command_stamped.hpp>
-#include <autoware_vehicle_msgs/msg/shift.hpp>
 #include <autoware_vehicle_msgs/msg/steering.hpp>
```

Comment:
In autoware.auto, VehicleKinematicState seems to be used for getting steering angle.

https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/blob/master/src/control/trajectory_follower_nodes/src/lateral_controller_node.cpp#L159-L162